TouchEffectSample.Android/TouchEffectSample.Android.csproj

```diff
@@ -119,5 +118,5 @@
       <Name>TouchEffect</Name>
     </ProjectReference>
   </ItemGroup>
- <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-</Project>
```

Comment:
the file should be discarded

Reply:
Once again, think I'm missing something here? Do you mean the `TouchEffectSample.Android.csproj` file?